docs/changelog.rst

```diff
@@ -1,6 +1,18 @@
 Release History
 ===============
 
+v0.21.0
+-------
+
+**Disclaimer**
+
+All releases in the v0.x.x series are subject to breaking changes from one version to another. After the release of v1.0.0, this project will be subject to `semantic versioning <http://semver.org/>`_.
```

Comment:
Are there breaking changes now?

Reply:
The plan details are a bit different now, and the images and dose slices are in a different coordinate space (DICOM versus IEC), so technically, yes, but this is the disclaimer that we add to all releases in the v0.x.x series.